crates/apollo-parser/src/parser/grammar/value.rs

```diff
@@ -339,4 +339,19 @@ query GraphQuery($graph_id: ID!, $variant: String) {
 
         assert!(ast.errors().next().is_none());
     }
+
+    #[test]
+    fn it_parse_mutation_with_escaped_chars_and_without() {
+        let input = r#"mutation {
+            createStore(draft: {
+              name: [{ locale: "en", value: "my \a store" }]
+            }) {
+              name(locale: "en")
+            }
+          }"#;
+        let parser = Parser::new(input);
+        let ast = parser.parse();
+
+        assert!(ast.errors().next().is_none());
```

Comment:
the parser should return an error because `\a` is not a valid character https://spec.graphql.org/June2018/#sec-String-Value

Reply:
Also fixed in this PR